From 6077588e8e9a9d59ca4e9d133605b778760fca71 Mon Sep 17 00:00:00 2001
From: Takamasa Takenaka <takamasa.takenaka@windriver.com>
Date: Tue, 5 Oct 2021 11:52:18 -0300
Subject: [PATCH 2/2] install resizepart.sh script

- resizepart.sh is a script without needing to handle command
  prompts interactively (clean copied from CentOS)
- Disable device-mapper for configure

Signed-off-by: Takamasa Takenaka <takamasa.takenaka@windriver.com>
---
 debian/parted.install | 1 +
 debian/rules          | 8 ++------
 2 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/debian/parted.install b/debian/parted.install
index 88efc46..4c4dccb 100644
--- a/debian/parted.install
+++ b/debian/parted.install
@@ -1,3 +1,4 @@
 sbin
 usr/share/man/man8/parted.8
 usr/share/man/man8/partprobe.8
+resizepart.sh usr/sbin
diff --git a/debian/rules b/debian/rules
index 590d032..f6ca9a0 100755
--- a/debian/rules
+++ b/debian/rules
@@ -15,12 +15,8 @@ export DEB_BUILD_MAINT_OPTIONS := hardening=+all
 CONFFLAGS := --sbindir=/sbin --enable-mtrace --enable-shared
 CONFFLAGS += --disable-gcc-warnings
 
-# Enable device-mapper only on Linux
-ifeq (linux, $(DEB_HOST_ARCH_OS))
-  CONFFLAGS += --enable-device-mapper
-else
-  CONFFLAGS += --disable-device-mapper
-endif
+# Disable device-mappaer
+CONFFLAGS += --disable-device-mapper
 
 # Everything above here is common to the deb and udeb builds.
 CONFFLAGS_UDEB := $(CONFFLAGS)
-- 
2.25.1

